### ISSUE TITLE
Add licensecheck.txt to review after successful build

### ISSUE THREAD

**Issue by pemensik** [FrostyX/fedora-review-service#17]
I think one of important things done by fedora-review is licensecheck.txt. I think it would be nice to have that file attached automatically after successful build in the service.

**Comment by FrostyX** [comment]
I agree, `licensecheck.txt` is very important. But there are multiple attachments produced by `fedora-review` and I don't want to treat `licensecheck.txt` any differently than the rest of them. So my plan is the following, and please let me know your thoughts.

There is an RFE for `copr-cli` https://github.com/fedora-copr/copr/issues/2459 that would allow us to run

```
copr-cli download-build 123456 --review
```

to download all the `fedora-review` results.  So the Fedora Review Service could say something like this within the Bugzilla comment:

```
Review template:
https://download.copr.fedorainfracloud.org/results/foo/bar/full/link/to/review.txt

All review files:
copr-cli download-build 123456 -r fedora-rawhide-x86_64 --review --spec

Please take a look if any issues were found.
```

**Comment by pemensik** [comment]
I think licensecheck.txt has special place in the review. Even review.txt output suggests that is the bug attachment, which should be attached to the review bug. It may be too long to be included in normal bugzilla comment, unlike rpmlint output. But it is not binary artefact like other results I think it makes it worthy to stay in the bug even after it is finished. It should be more simple to click to it from the review bug.

*No*, I do not think downloading it with build together with build artefacts would help. The point is to spare user to having it uploaded to bug by hand, but make it attached anyway. I think only files.dir might be handled the same way as licensecheck.txt. Other artifacts might be obtained from copr instance.

**Comment by pemensik** [comment]
I think review artefacts would have to be deleted after some time. But I would like to licensecheck stay in the review forever. So even link to package build would not be ideal. That is why I think it should be an attachment.

**Comment by FrostyX** [comment]
> But I would like to licensecheck stay in the review forever.

The `licensecheck.txt` for the last build in a Copr project (and therefore for each bugzilla review ticket) will stay available forever. For example [here we have 10 months old build](https://copr.fedorainfracloud.org/coprs/g/fedora-review/fedora-review-2158954-python-term-background/build/5205691/) and [its licensecheck.txt is still there](https://download.copr.fedorainfracloud.org/results/@fedora-review/fedora-review-2158954-python-term-background/fedora-rawhide-x86_64/05205691-python-term-background/fedora-review/licensecheck.txt).

IMHO there is no need for having historical licensecheck output at hand, so I am still leaning towards the 

```
All review files:
copr-cli download-build 123456 -r fedora-rawhide-x86_64 --review --spec
```

I am trying to keep the number of network requests while handling each fedora messaging message to a minimum.
